lib/chewy/type/import.rb

```diff
@@ -66,6 +70,7 @@ def bulk options = {}
           suffix = options.delete(:suffix)
           bulk_size = options.delete(:bulk_size)
           body = options.delete(:body)
+          journal = options.delete(:journal)
```

Comment:
I think so because we use `options` below we're merging it with `bulk_body` and `header` and we definitely shouldn't provide some "garbage" to `Chewy.client.bulk`